datashader/__init__.py

```diff
@@ -1,6 +1,15 @@
 from __future__ import absolute_import
 
-__version__ = '0.6.5'
+try:
+    from param.version import Version
+    __version__ = str(Version(fpath=__file__, archive_commit="$Format:%h$", reponame="datashader"))
+    del Version
+except:
+    import os, json
+    __version__ = json.load(open(os.path.join(os.path.split(__file__)[0],
+                                              '.version'), 'r'))['version_string']
+    del os
+    del json
```

Comment:
I'm not sure why you want to use ``del``  here. Especially for ``os`` and ``json`` which are common imports from the standard lib...

Reply:
Seems like that's optional, but it's nice to tidy up the namespace and not leave incidentals like that hanging round to confuse tab complete and dir()...